Assets/MixedRealityToolkit.Examples/Experimental/HandMenuLayout/Materials/HandMenu_ToggleDialog_Backplate.mat

```diff
@@ -0,0 +1,191 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!21 &2100000
+Material:
+  serializedVersion: 6
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_Name: HandMenu_ToggleDialog_Backplate
```

Comment:
Why does the `HolographicBackPlate` material not work here?

Reply:
Removed! I applied the backing material from the hand menus instead. The reasoning was that I wanted to tweak some of the material's parameters without breaking HoloGraphicBackPlate across the rest of the project.